docs/src/user_docs/stdlib/collections.md

```diff
@@ -0,0 +1,22 @@
+# Collections
+Namespace `std::crypto` contains modules for commonly authenticated data structures.
+
+## Merkle Mountain Range
+Module `std::collections::mmr` contains procedures for manipulating [Merkle Mountain Ranges](https://github.com/opentimestamps/opentimestamps-server/blob/master/doc/merkle-mountain-range.md) data structure which can be used as an append-only log.
+
+| Procedure | Description |
+| ----------- | ------------- |
+| get | Loads the leaf at the absolute `pos` in the MMR onto the stack.<br /><br />This MMR implementation supports only u32 positions.<br /><br />Inputs:<br />- Operand stack: [pos, mmr_ptr, ...]<br /><br />Output:<br />- Operand stack: [N, ...]<br /><br />Where `N` is the leaf loaded from the MMR located starting at memory address `mmr_ptr`. |
```

Comment:
nit: I would change "from the MMR located starting at memory address `mmr_ptr`" to "from the MMR whose memory location starts at `mmr_ptr`". I would make the same edit below for `add`

Reply:
Does `This MMR implementation supports only u32 positions.` mean, `get` only accepts positions in the u32 format or only `std::u32::MAX` positions?

Reply:
The positions are in the range $[0 .. 2^{32})$ - I'll update the comment to say this explicitly.